sdk/authorization/Azure.ResourceManager.Authorization/src/autorest.md

```diff
@@ -0,0 +1,124 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+
+azure-arm: true
+csharp: true
+tag: package-2020-10-01
+library-name: Authorization
+namespace: Azure.ResourceManager.Authorization
+require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/a416080c85111fbe4e0a483a1b99f1126ca6e97c/specification/authorization/resource-manager/readme.md
+output-folder: Generated/
+clear-output-folder: true
+skip-csproj: true
+modelerfour:
```

Comment:
Can we remove this configuration? I think it's better to use renaming to avoid the name conflicts as we are about to have this package GA soon.